<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="cardboard_box">
    <pose>0 0 0.075 0 0 0</pose> <!-- Adjust pose to match new size -->
    <link name="link">
      <inertial>
        <mass>2</mass>
        <inertia>
          <ixx>0.041666667</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.056666667</iyy>
          <iyz>0</iyz>
          <izz>0.068333333</izz>
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry>
          <box>
            <size>0.25 0.2 0.15</size> <!-- Reduced dimensions -->
          </box>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>1.0</mu>
              <mu2>1.0</mu2>
            </ode>
          </friction>
          <contact>
            <ode>
              <kp>10000000.0</kp>
              <kd>1.0</kd>
              <min_depth>0.001</min_depth>
              <max_vel>0.1</max_vel>
            </ode>
          </contact>
        </surface>
      </collision>
      <visual name="visual">
        <pose>0 0 -0.075 0 0 0</pose> <!-- Adjust pose to match new size -->
        <geometry>
          <mesh>
            <uri>model://cardboard_box/meshes/cardboard_box.dae</uri>
            <scale>0.62965948 0.503727584 0.377795688</scale> <!-- Reduced scale -->
          </mesh>
        </geometry>
      </visual>
    </link>
  </model>
</sdf>